week0003/truongngodang/java/Week0003.java

```diff
@@ -0,0 +1,154 @@
+import java.util.*;
+
+public class Week0003 {
+
+    private static final String ADD_NEW_CLIENT = "A";
+    private static final String DEPOSIT_MONEY = "D";
+    private static final String WITHDRAW_MONEY = "W";
+
+    public static void main(String[] args) {
+
+        Scanner scan = new Scanner(System.in);
+        HashMap<String, Client> clients = new HashMap<String, Client>();
+        ArrayList<String> output = new ArrayList<String>();
+
+        for (int i = 0; scan.hasNext(); i++) {
+            processRequest(new ProcessStringRequest(scan.nextLine()), clients, output);
+        }
+        scan.close();
+
+        output.forEach(System.out::println);
```

Comment:
You should print inside the loop too. Now you buffer the output and if you are running a server, you can't buffer forever ;)